modules/op-installing-pipelines-operator-using-the-cli.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * openshift_pipelines/installing-pipelines.adoc
+
+[id="op-installing-pipelines-operator-using-the-cli_{context}"]
+= Installing the OpenShift Pipelines Operator using the CLI
+
+Instead of using the {product-title} Web Console, you can install {pipelines-title} Operator from the OperatorHub using the CLI.
+
+[discrete]
+.Procedure
+
+. Create a Subscription object YAML file to subscribe a namespace to the {pipelines-title} Operator,
+for example, `sub.yaml`:
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-pipelines-operator
+  namespace: openshift-operators
+spec:
+  channel:  <channel name> <1>
+  name: <operator-name> <2>
```

Comment:
I think that you can hard-code `openshift-pipelines-operator`. @vdemeester can you confirm?

Reply:
@ppitonak yes the operator-name can be hardcoded `openshift-pipelines-operator`

Reply:
done